aws/rand-password/main.tf

```diff
@@ -0,0 +1,49 @@
+
+locals {
+  password_path = "/${var.environment}/${var.service_name}/${var.keyname}"
```

Comment:
This is pretty opinionated, could we put it in the readme that this is the path it will live at? And/or return the path as an output?

Reply:
I'll do you one better and make and add an input variable to give you the option to create the path but defaults to the generated path if no input variable is set

Also outputs the password path 

Fixed in 364b7130853fb808acaf931d9d5c69db61621d0f